estate/tests/test_estate_property.py

```diff
@@ -0,0 +1,116 @@
+from odoo.tests.common import TransactionCase
+from odoo.exceptions import UserError
+from odoo.tests import tagged, Form
+
+
+ADMINISTRATOR_ID = 3
```

Comment:
Data created in xml/csv have an external_id. This is the id of the xml record. It can be easily found. For example here you can search `model="res.partner"' in all your xml files and then you can check the demo data. 

When you need to cache a record by its id you can used the ref method of self.env. It contacts the ir_model_data table which contains all the external ids and with the other information of the record in that table its able to find where is the record asked. 

Also we usually do not write global constants. So you could just put it inside the setUpClass.

```suggestion
ADMINISTRATOR_ID = self.env.ref("base.partner_admin").id
```